### PR TITLE
Changed clear varnish cache to clear all domains in the environments

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
@edx/pipeline-team @clintonb Please review

### CODE REVIEW

**Comment by clintonb** [comment]
👍

File: tubular/drupal.py [hunk 1]
```diff
@@ -100,24 +113,26 @@ def fetch_deployed_tag(env, username, password, path_name):
...
     Raises:
         KeyError: Raised if env value is invalid.
     """
     api_client = get_api_client(username, password)
-    domain = VALID_ENVIRONMENTS[env]
-    response = api_client.delete(CLEAR_CACHE_URL.format(env=env, domain=domain))
-    response_json = parse_response(response, "Failed to clear cache.")
-    return check_state(response_json["id"], username, password)
+    domains = VALID_ENVIRONMENTS[env]
+    for domain in domains:
```
Comment (macdiesel) [inline tubular/drupal.py]: Just so I am clear, if one domain fails, the remaining domains should not be cleared? Would it be better here to continue and try to clear as many domains as possible?

Perhaps attempt them all and still raise an exception listing those that failed? 

My concern here is that you could leave these caches in a partially cleared state.

**Comment by Dillon-Dumesnil** [comment]
@macdiesel Updated to continue through all domains despite an error and to raise an error at the end.

**Comment by macdiesel** [comment]
👍

**Pull request merged**